src/components/core/TruncatedText/TruncatedText.js

```diff
@@ -1,19 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
+import { ellipsis } from 'polished';
 
 const Base = ({ is: Component, ...props }) => <Component {...props} />;
 
 /**
  * Truncate text with an ellipsis.
  */
-const TruncatedText = styled(Base)`
-	white-space: nowrap;
-	overflow: hidden;
-	text-overflow: ellipsis;
+const TruncatedText = styled(Base).attrs({
+	title: props => props.children,
```

Comment:
Depending on type of children this can become `[Object object]`. I think you have to type check.

Reply:
Yeah, we can throw if `children` isn't string and `title` isn't specified.